core/commands/id.go

```diff
@@ -19,7 +19,7 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
-const offlineIdErrorMessage = `ID command fails when run without daemon, we are working to fix this.
+const offlineIdErrorMessage = `Id command fails when run without the daemon; we are working to fix this.
```

Comment:
huh, this is no longer true. You can run `ipfs id` without a running daemon just fine